explain-views.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Views
```

Comment:
The title is duplicated with the title of https://docs.pingcap.com/tidb/dev/views. What about using another title?

Reply:
Is there a convention to incorporate path into the title, such as `Explain -> Views`?

Reply:
`Explain -> Views` does not clearly indicate the content of this document. What about `Views in the Result of EXPLAIN`?

Reply:
I need to use the same convention with the following content:
```
  Overview
  Explain Walkthough
  Indexes
  Joins
  Subqueries
  Views
  Partitions
```

I will go with `EXPLAIN Statements Using Views`.

Reply:
`EXPLAIN Statements Using Views` LGTM. Could you make the titles of the list more specific in a similar way?

Reply:
Yes, I am using the same convention. For example: https://github.com/pingcap/docs/pull/3922